website/docs/r/automation_credential.html.markdown

```diff
@@ -0,0 +1,67 @@
+---
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_automation_credential"
+sidebar_current: "docs-azurerm-resource-automation-credential"
+description: |-
+  Creates a new Automation Credential.
+---
+
+# azurerm\_automation\_credential
+
+Creates a new Automation Credential.
+
+## Example Usage
+
+```
+resource "azurerm_resource_group" "example" {
+ name = "resourceGroup1"
+ location = "West Europe"
+}
+
+resource "azurerm_automation_account" "example" {
+  name                = "account1"
+  location            = "${azurerm_resource_group.example.location}"
+  resource_group_name = "${azurerm_resource_group.example.name}"
+  sku {
+        name = "Free"
+  }
+}
+
+resource "azurerm_automation_credential" "example" {
+  name                = "credential1"
+  resource_group_name = "${azurerm_resource_group.example.name}"
+  account_name        = "${azurerm_automation_account.example.name}"
+  username           = "example_user"
+  password            = "example_pwd"
+  description         = "This is an example credential"
```

Comment:
can we add in the missing `}` at the end of this example?

Reply:
Done